pkg/repository/popular/docker/popular.go

```diff
@@ -7,15 +7,16 @@ import (
 	"fmt"
 	"log"
 	"net/http"
+	"strings"
 	"time"
 
 	"github.com/aquasecurity/binfinder/pkg/model"
 	"github.com/aquasecurity/binfinder/pkg/repository/popular"
 )
 
 const (
-	dockerAPI    = "https://hub.docker.com/api/content/v1/products/search?image_filter=official&page=%v&page_size=100&q=&type=image"
-	dockerTagAPI = "https://hub.docker.com/v2/repositories/library/%v/tags?page=1&page_size=1"
+	dockerAPI    = "https://hub.docker.com/v2/repositories/library/?page=1&page_size=100"
+	dockerTagAPI = "https://hub.docker.com/v2/repositories/library/%v/tags"
```

Comment:
@rahul2393 - I noticed these don't give more than 2 pages worth of results as the Next link is null in page 2:

./binfinder -top=4000 -all-tags
2020/12/10 16:47:20 Fetching page:  https://hub.docker.com/v2/repositories/library/?page=1&page_size=100
2020/12/10 16:47:21 Found images:  100
2020/12/10 16:47:32 Fetching page:  https://hub.docker.com/v2/repositories/library/?page=2&page_size=100
2020/12/10 16:47:32 Found images:  64